rules/helm/delete.tmpl.rb

```diff
@@ -9,4 +9,6 @@
 
 args = [helm, 'delete', install_name, '--namespace', namespace]
 
+puts "CMD #{args}"
```

Comment:
```suggestion
puts "+ #{args}"
```

Reply:
Sorry. I am not taking the suggestion. The little `+` does not highlight the command enough for me, IMHO.